app/controllers/urls_controller.rb

```diff
@@ -0,0 +1,76 @@
+class UrlsController < ApplicationController
+  require 'securerandom'
+
+  before_action :load_url, only: [:show, :update]
+  #before_action :generate_slug, :short_url, only: :create
+
+  # GET /urls or /urls.json
```

Comment:
Remove redundant comments like these. Rails has conventions, by which it is clear that the `index` action corresponds to the `GET /urls` path only.